cvat/apps/consensus/consensus_reports.py

```diff
@@ -0,0 +1,493 @@
+        data=task_report["data"],
+        consensus_score=task_report["consensus_score"],
+        assignee=task_report["assignee"],
+    )
+    db_task_report.save()
+
+    db_job_reports = []
+    for job_report in job_reports:
+        db_job_report = ConsensusReport(
+            task=task_report["task"],
+            job=job_report["job"],
+            target_last_updated=job_report["target_last_updated"],
+            data=job_report["data"],
+            consensus_score=job_report["consensus_score"],
+            assignee=job_report["assignee"],
+        )
+        db_job_reports.append(db_job_report)
+
+    db_job_reports = bulk_create(db_model=ConsensusReport, objects=db_job_reports, flt_param={})
+
+    for assignee, assignee_info in assignee_report_data.items():
+        # db_assignee = models.User.objects.get(id=)
+        db_assignee_report = AssigneeConsensusReport(
+            task=task_report["task"],
+            consensus_score=np.round(100 * assignee_info["consensus_score"]),
+            conflict_count=assignee_info["conflict_count"],
+            assignee=assignee,
+            consensus_report_id=db_task_report.id,
+        )
+        db_assignee_report.save()
+
+    db_conflicts = []
+    db_report_iter = itertools.chain([db_task_report], db_job_reports)
+    report_iter = itertools.chain([task_report], job_reports)
+    for report, db_report in zip(report_iter, db_report_iter):
+        if not db_report.id:
+            continue
+        for conflict in report["conflicts"]:
+            db_conflict = ConsensusConflict(
+                report=db_report,
+                type=conflict["type"],
+                frame=conflict["frame_id"],
+            )
+            db_conflicts.append(db_conflict)
+
+    db_conflicts = bulk_create(db_model=ConsensusConflict, objects=db_conflicts, flt_param={})
+
+    db_ann_ids = []
+    db_conflicts_iter = iter(db_conflicts)
+    for report in itertools.chain([task_report], job_reports):
+        for conflict, db_conflict in zip(report["conflicts"], db_conflicts_iter):
+            for ann_id in conflict["annotation_ids"]:
+                db_ann_id = models.AnnotationId(
+                    conflict=db_conflict,
+                    job_id=ann_id["job_id"],
+                    obj_id=ann_id["obj_id"],
+                    type=ann_id["type"],
+                    shape_type=ann_id["shape_type"],
+                )
+                db_ann_ids.append(db_ann_id)
+
+    db_ann_ids = bulk_create(db_model=models.AnnotationId, objects=db_ann_ids, flt_param={})
+
+    return db_task_report.id
+
+
+def prepare_report_for_downloading(db_report: ConsensusReport, *, host: str) -> str:
+    # copied from quality_reports.py
+    # Decorate the report for better usability and readability:
+    # - add conflicting annotation links like:
+    # <host>/tasks/62/jobs/82?frame=250&type=shape&serverID=33741
+    # - convert some fractions to percents
+    # - add common report info
+
```

Comment:
In quality reports, there was assignee info added. It might make sense to include it here as well.

Reply:
assignee info is in `QualityReport` similar to what we have in `ConsensusReport` but not in `AnnotationConflict`. Am I mistaking something for something else?

Reply:
I'm speaking about [this function](https://github.com/cvat-ai/cvat/blob/a69e1228ac8ab120ca9f1274348e83ddcb89d4be/cvat/apps/quality_control/quality_reports.py#L2490-L2498) for assignee formatting in the report output. Please use it here for consistent representation of assignee in these two kinds of reports.